pkg/catalog/catalog.go

```diff
@@ -1770,29 +1771,37 @@ type GCUncommittedMark struct {
 }
 
 type PrepareGCUncommittedInfo struct {
-	Mark     *GCUncommittedMark
-	Metadata graveler.GarbageCollectionRunMetadata
+	RunID      string `json:"run_id"`
+	Location   string `json:"location"`
+	ObjectName string `json:"object_name"`
```

Comment:
```suggestion
	BasePath   string `json:"base_path"`
	Filename string `json:"filename"`
```
"object" is more correct, but less comprehensible.

Reply:
You update location to base_path - it is the same as gc metadata parameters. Do you still think we need to break alignment/

Reply:
Not really - the object name was mostly what bothered me.